flytekit/core/utils.py

```diff
@@ -211,6 +211,29 @@ def write_proto_to_file(proto, path):
         writer.write(proto.SerializeToString())
 
 
+def get_extra_headers_for_signed_url(signed_url):
```

Comment:
Added helper here rather than as a static method in FlyteRemote, this is so that both FlyteRemote, FlyteFS can use the helper.